### ISSUE TITLE
7.2.5 Error

### ISSUE THREAD

**Issue by swinu** [phanx-wow/Bugger#3]
Been getting this since 7.2.5

```
33x Bugger\Bugger-7.1.0.0.lua:293: attempt to index field 'title' (a nil value)
Bugger\Bugger-7.1.0.0.lua:293: in function `ShowError'
Bugger\Bugger-7.1.0.0.lua:339: in function `ShowSession'
Bugger\Bugger-7.1.0.0.lua:348: in function <Bugger\Bugger.lua:344>
(tail call): ?
[C]: ?
[C]: in function `ChatFrame_OnHyperlinkShow'
[string "*:OnHyperlinkClick"]:1: in function <[string "*:OnHyperlinkClick"]:1>

Locals:
self = <table> {
 BugGrabber_BugGrabbed = <function> defined @Bugger\Bugger.lua:160
 previous = <unnamed> {
 }
 UnregisterAllEvents = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:101
 ToggleFrame = <function> defined @Bugger\Bugger.lua:344
 ShowError = <function> defined @Bugger\Bugger.lua:256
 dataObject = <table> {
 }
 next = <unnamed> {
 }
 first = 239
 GetName = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:43
 ShowSession = <function> defined @Bugger\Bugger.lua:320
 FormatLocals = <function> defined @Bugger\Bugger.lua:225
 OnLogin = <function> defined @Bugger\Bugger.lua:98
 FormatError = <function> defined @Bugger\Bugger.lua:244
 showLocals = <unnamed> {
 }
 Print = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:59
 last = 242
 session = "current"
 PRINT_PREFIX = "|cff00ddbaBugger:|r"
 db = <table> {
 }
 L = <table> {
 }
 error = 242
 scrollFrame = BuggerFrameScrollFrame {
 }
 clear = <unnamed> {
 }
 reload = <unnamed> {
 }
 InitializeDB = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:125
 OnLoad = <function> defined @Bugger\Bugger.lua:92
 RegisterEvent = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:82
 menu = BuggerMenu {
 }
 indexLabel = <unnamed> {
 }
 editBox = TopText {
 }
 FormatStack = <function> defined @Bugger\Bugger.lua:198
 IsEventRegistered = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:105
 tabs = <table> {
 }
 lastError = 432228.688
 frame = BuggerFrame {
 }
 GetNumErrors = <function> defined @Bugger\Bugger.lua:122
 UnregisterEvent = <function> defined @Bugger\Libs\PhanxAddonStub\AddonStub.lua:96
 SetupFrame = <function> defined @Bugger\Bugger.lua:354
}
index = 242
showLocals = nil
errors = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
 9 = <table> {
 }
 10 = <table> {
 }
 11 = <table> {
 }
 12 = <table> {
 }
 13 = <table> {
 }
 14 = <table> {
 }
 15 = <table> {
 }
 16 = <table> {
 }
 17 = <table> {
 }
 18 = <table> {
 }
 19 = <table> {
 }
 20 = <table> {
 }
 21 = <table> {
 }
 22 = <table> {
 }
 23 = <table> {
 }
 24 = <table> {
 }
 25 = <table> {
 }
 26 = <table> {
 }
 27 = <table> {
 }
 28 = <table> {
 }
 29 = <table> {
 }
 30 = <table> {
 }
 31 = <table> {
 }
 32 = <table> {
 }
 33 = <table> {
 }
 34 = <table> {
 }
 35 = <table> {
 }
 36 = <table> {
 }
 37 = <table> {
 }
 38 = <table> {
 }
 39 = <table> {
 }
 40 = <table> {
 }
 41 = <table> {
 }
 42 = <table> {
 }
 43 = <table> {
 }
 44 = <table> {
 }
 45 = <table> {
 }
 46 = <table> {
 }
 47 = <table> {
 }
 48 = <table> {
 }
 49 = <table> {
 }
 50 = <table> {
 }
 51 = <table> {
 }
 52 = <table> {
 }
 53 = <table> {
 }
 54 = <table> {
 }
 55 = <table> {
 }
 56 = <table> {
 }
 57 = <table> {
 }
 58 = <table> {
 }
 59 = <table> {
 }
 60 = <table> {
 }
 61 = <table> {
 }
 62 = <table> {
 }
 63 = <table> {
 }
 64 = <table> {
 }
 65 = <table> {
 }
 66 = <table> {
 }
 67 = <table> {
 }
 68 = <table> {
 }
 69 = <table> {
 }
 70 = <table> {
 }
 71 = <table> {
 }
 72 = <table> {
 }
 73 = <table> {
 }
 74 = <table> {
 }
 75 = <table> {
 }
 76 = <table> {
 }
 77 = <table> {
 }
 78 = <table> {
 }
 79 = <table> {
 }
 80 = <table> {
 }
 81 = <table> {
 }
 82 = <table> {
 }
 83 = <table> {
 }
 84 = <table> {
 }
 85 = <table> {
 }
 86 = <table> {
 }
 87 = <table> {
 }
 88 = <table> {
 }
 89 = <table> {
 }
 90 = <table> {
 }
 91 = <table> {
 }
 92 = <table> {
 }
 93 = <table> {
 }
 94 = <table> {
 }
 95 = <table> {
 }
 96 = <table> {
 }
 97 = <table> {
 }
 98 = <table> {
 }
 99 = <table> {
 }
 100 = <table> {
 }
 101 = <table> {
 }
 102 = <table> {
 }
 103 = <table> {
 }
 104 = <table> {
 }
 105 = <table> {
 }
 106 = <table> {
 }
 107 = <table> {
 }
 108 = <table> {
 
```

**Comment by nmsmith22389** [comment]
I get the same :/

**Comment by ckaotik** [comment]
In case anyone else stumbles upon this: This can be fixed by using a capital "T" in line 534 of `Bugger.lua`:
`self.title       = f.Title`

**Comment by Phanx** [comment]
Fixed in a8837a3
